src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -416,46 +416,33 @@ private bool CanUseTupleTypeName(INamedTypeSymbol tupleSymbol)
             return true;
         }
 
-        private static bool HasNonDefaultTupleElementNames(INamedTypeSymbol tupleSymbol)
+        private static bool HasNonDefaultTupleElements(INamedTypeSymbol tupleSymbol)
```

Comment:
Consider moving to `ISymbolExtensions`, beside `IsDefaultTupleElement`, and share with VB.